src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -143,6 +140,47 @@ const MetaTraderUI = (() => {
         return num_servers;
     };
 
+    const resetMT5TradingPassword = () => BinarySocket.send({
+        verify_email: Client.get('email'),
+        type        : 'trading_platform_mt5_password_reset',
+    });
+
+    const populatePasswordManager = () => {
+        const $mng_passwd  = $container.find('#frm_manage_password');
+        const $step_1 = $mng_passwd.find('.step-1');
+        const $step_2 = $mng_passwd.find('.step-2');
+        const $button = $mng_passwd.find('#password_change_button');
+        const $confirm_button = $mng_passwd.find('#password_change_confirm_buttons .btn_ok');
+        const $cancel_button = $mng_passwd.find('#password_change_confirm_buttons .btn_cancel');
+        const setStep = (step) => {
+            switch (step) {
+                case 1:
```

Comment:
(Suggestion) 
Possible to map these numbers in a constant object for a better understanding of the current step? e.g.:
```
case STEPS.SERVER_SELECTION: 
```